recipe_scrapers/number2pencil.py

```diff
@@ -0,0 +1,74 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class Number2Pencil(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "number-2-pencil.com"
+
+    def author(self):
+        author_from_schema = self.schema.author()
+        if author_from_schema:
+            return author_from_schema
+
+        author_span = self.soup.find("span", {"class": "entry-author"})
+        author_name = (
+            author_span.find("span", {"class": "entry-author-name"})
+            if author_span
+            else None
+        )
+        return author_name.text if author_name else "number-2-pencil"
+
+    def category(self):
+        category_from_schema = self.schema.category()
+        if category_from_schema:
+            return category_from_schema
+
+        entry_categories_div = self.soup.find("span", {"class": "entry-categories"})
+        categories = [a.text for a in entry_categories_div.find_all("a", href=True)]
+        return ", ".join(categories)
+
+    def description(self):
+        description_from_schema = self.schema.description()
+        if description_from_schema:
+            return description_from_schema
+
+        entry_content_div = self.soup.find("div", {"class": "entry-content"})
+        return entry_content_div.find("em").text
+
+    def cuisine(self):
+        cuisine_from_schema = self.schema.cuisine()
+        if cuisine_from_schema:
```

Comment:
What situation(s) is this `if` condition intended to handle?  I took a quick look at the unit tests but it wasn't clear to me.

Reply:
Ah, OK - see what you mean.  Test 1 produces the following result when the `cuisine` method is called:

```python
recipe_scrapers._exceptions.SchemaOrgException: recipe-scrapers exception: No cuisine data in SchemaOrg.
```

That's expected, and in fact good: it's the appropriate [`recipe-scrapers` exception](https://github.com/hhursev/recipe-scrapers/blob/6222f1fe610aa445b074d7be12d205fc2409b88c/recipe_scrapers/_exceptions.py) that should be thrown to tell the caller that the requested data couldn't be found on the page.

An uncaught exception causes the current method to exit immediately, so the `if` block will not be evaluated when that happens, and so we can simplify the method to:

```
    def cuisine(self):
        return self.schema.cuisine()
```